hypervisor/mmu.c

```diff
@@ -0,0 +1,37 @@
+#define MAX_PAGE_PDE    8
```

Comment:
I'm not sure, is it correct that there are only 8 pages? In https://wiki.osdev.org/Setting_Up_Paging they used 1024

Reply:
too big. 1024 will be too big.